packages/block-editor/src/components/block-card/stories/index.story.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { box, button, cog, paragraph } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import BlockCard from '../';
+
+const meta = {
+	title: 'BlockEditor/BlockCard',
+	component: BlockCard,
+	parameters: {
+		docs: {
+			description: {
+				component:
+					'The `BlockCard` component allows to display a "card" which contains the title of a block, its icon and its description.',
+			},
+			canvas: { sourceState: 'shown' },
+		},
+	},
+	argTypes: {
+		title: {
+			control: 'text',
+			description: 'The title of the block.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		description: {
+			control: 'text',
+			description: 'A description of the block functionality.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		icon: {
+			control: 'select',
+			options: [ 'paragraph', 'cog', 'box', 'button' ],
+			mapping: {
+				paragraph,
+				cog,
+				box,
+				button,
+			},
+			description: 'The icon to display for the block.',
```

Comment:
```suggestion
			description:
				"The icon of the block. This can be any of [WordPress' Dashicons](https://developer.wordpress.org/resource/dashicons/), or a custom `svg` element.",
			table: {
				type: { summary: 'string | object' },
			},
```

It's a good idea to add a link to the Dashicon page and the correct type as defined in the README.

![image](https://github.com/user-attachments/assets/25bf1cc7-e357-48c7-ad62-33440e4d721e)

Reply:
Hi @t-hamano,
Thank you for the suggestion I have implemented the changes.